datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/components/MenuColumn.tsx

```diff
@@ -37,6 +44,28 @@ export default function MenuColumn({ field }: Props) {
                             <ImpactAnalysisIcon /> &nbsp; See Column Lineage
                         </MenuItem>
                     </Menu.Item>
+                    {navigator.clipboard && (
+                        <Menu.Item key="1">
+                            <MenuItem
+                                onClick={() => {
+                                    navigator.clipboard.writeText(field.fieldPath);
+                                }}
+                            >
+                                <CopyOutlinedIcon /> &nbsp; Copy Column Field Path
+                            </MenuItem>
+                        </Menu.Item>
+                    )}
+                    {navigator.clipboard && (
+                        <Menu.Item key="2">
+                            <MenuItem
+                                onClick={() => {
+                                    navigator.clipboard.writeText(urn);
```

Comment:
I'm afraid that this is going to copy the entity urn not the column urn.

Reply:
Yes. Using `generateSchemaFieldUrn` function to get the URN of the column.